File: docs/examples/python/connect_example.py
import taos

def create_connection():
    # all parameters are optional.
    conn = None
    host = "localhost"
    port = 6031
    try:
        conn = taos.connect(
            user="root",
            password="taosdata",
            host=host,
            port=port,
        )
        print(f"Connected to {host}:{port} successfully.");
    except Exception as err:
        print(f"Failed to connect to {host}:{port} , ErrMessage:{err}")
        raise err
    finally:
        if conn:
            conn.close()


if __name__ == "__main__":
    create_connection()

File: docs/examples/c-ws-new/connect_example.c
// to compile: gcc -o connect_example connect_example.c -ltaos

#include <stdio.h>
#include <stdlib.h>
#include "taos.h"

int main() {
  const char *host = "localhost";
  const char *user = "root";
  const char *passwd = "taosdata";
  const char *db = NULL;
  uint16_t    port = 6041;

  int code = taos_options(TSDB_OPTION_DRIVER, "websocket");
  if (code != 0) {
    fprintf(stderr, "Failed to set driver option, code: %d\n", code);
    return -1;
  }

  TAOS *taos = taos_connect(host, user, passwd, db, port);
  fprintf(stdout, "Connected to %s:%hu successfully.\n", host, port);

  /* put your code here for read and write */

  taos_close(taos);
  taos_cleanup();
}
